.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+## What is the current behavior?
+
+<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
```

Comment:
@malker97 The issue itself can provide current behavior and new behavior, I only need the issue number and pull request type here.